martin-mbtiles/src/tile_copier.rs

```diff
@@ -382,29 +398,21 @@ pub async fn apply_mbtiles_diff(
         .execute(&mut conn)
         .await?;
 
-    query(
-        "
+    let sql = "
     DELETE FROM tiles 
     WHERE (zoom_level, tile_column, tile_row) IN 
-        (SELECT zoom_level, tile_column, tile_row FROM diffDb.tiles WHERE tile_data ISNULL);",
-    )
-    .execute(&mut conn)
-    .await?;
-
-    query(
-        "INSERT OR REPLACE INTO tiles (zoom_level, tile_column, tile_row, tile_data) 
-    SELECT * FROM diffDb.tiles WHERE tile_data NOTNULL;",
-    )
-    .execute(&mut conn)
-    .await?;
+        (SELECT zoom_level, tile_column, tile_row FROM diffDb.tiles WHERE tile_data ISNULL)";
+    query(sql).execute(&mut conn).await?;
+
+    let sql = "INSERT OR REPLACE INTO tiles (zoom_level, tile_column, tile_row, tile_data) 
+        SELECT * FROM diffDb.tiles WHERE tile_data NOTNULL";
+    query(sql).execute(&mut conn).await?;
```

Comment:
nit: I don't think this change is necessary; I think it would be cleaner to just leave it as it was